### DISCUSSION TITLE
Weird behavior after Windows 11 22h2 update

### DISCUSSION THREAD

**Discussion by peterconcept** [artisan-roaster-scope/artisan#978] [General]
So my notebook has been updated to win11 22h2 and odd behavior in Artisan started at that point.
After update I was unable to control my Probat P12 2 from Artisan, had to reset Artisan to factory settings and basically renew all the settings from memory. When I used my saved setting backup I could not control roast. It is easy to repair not much of an issue.

Issues:
- Detection of the start is extremely slow - 10c off actual charge
- When I start the roast, put the gas on, gas goes to selected power setting, after a while - I am experiencing "ghost" gas changes - gas changes on the Artisan screen. It goes up few percent on the screen, gas on the machine stays the same as a value I have set. This goes on couple of times during the roast.

**Comment by MAKOMO** [comment]
Unclear how a system update could change an application setup. Maybe Microsoft knows. Anyhow, you forgot to indicate your Artisan version. The slow CHARGE detection was fixed and the fix will be released with the new version that appears later this month. The ghost gas setting is most likely caused by wrong event quantifier settings. Either attach your artisan-settings.aset file here (you have to rename it to artisan-settings.txt before drag-and-drop it here on a comment field) to allow us a quick inspection or just do another factory reset and re-run the "Probat 7" machine setup again to correct this.

**Reply by peterconcept** [reply]
Thank you for quick response. I am going to do another factory reset first and if it won't do the job, I'll post the settings. 
version 2.6.0.

I just wanted to warn users about W11 22h2 update, maybe it will help somebody.
